### PR TITLE
Add config for TFChain WebSocket in ctor 2

### PR DESCRIPTION
**Pull request by MohamedElmdary**
### Related Issues

- #29

### CODE REVIEW

File: lib/client.ts [hunk 1]
```diff
@@ -9,56 +9,56 @@ import { Buffer } from "buffer"
 import { sign, KPType } from './sign'
 import { v4 as uuidv4 } from 'uuid';
 import { getTwinFromTwinAddress, getTwinFromTwinID } from "./util";
-
-
+import { WsProvider, ApiPromise } from "@polkadot/api";
+import type { WebSocket as WSConnection } from "ws";
```
Comment (AhmedHanafy725) [inline lib/client.ts:13]: is this working on the browser? `ws` package is a nodejs package

File: lib/client.ts [hunk 2]
```diff
@@ -103,15 +104,18 @@ class Client {
                 window.onunload = this.disconnect;
             }
         } catch (err) {
-            if (this.con && this.con.readyState == this.con.OPEN) {
-                this.con.close()
+            const c = this.con as WSConnection;
+            if (c && c.readyState == c.OPEN) {
+              c.close();
```
Comment (AhmedHanafy725) [inline lib/client.ts:109]: why creating `c` here, it's already this.con?

File: lib/client.ts [hunk 3]
```diff
@@ -103,15 +104,18 @@ class Client {
...
             }
             throw new Error({ message: `Unable to connect due to ${err}` })
         }
 
     }
 
     disconnect() {
+        this.api?.disconnect();
+        this.api?.off("disconnected", this.__handleConnection);
```
Comment (AhmedHanafy725) [inline lib/client.ts]: I think we should remove the event handler before calling the disconnect method

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: lib/client.ts (hunk 1 above)
Comment (MohamedElmdary) [reply lib/client.ts:13]: it's `import type` will be removed in compile time

File: lib/client.ts (hunk 2 above)
Comment (MohamedElmdary) [reply lib/client.ts:109]: To assign a type for it as it's marked as `never` for some reason

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**